packages/nextjs-kit/src/__mocks__/image.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+export default function image(
+	props: React.DetailedHTMLProps<
+		React.ImgHTMLAttributes<HTMLImageElement>,
+		HTMLImageElement
+	>,
+) {
+	return <img {...props} />;
+}
```

Comment:
I was refactoring the ContentWithImage component and I don't think this mock is necessary. I'm not sure it is even doing what we are expecting it to because I get the same results with or without the mock.